examples/mobc/src/src_user/Settings/component_driver_super/driver_super_params.h

```diff
@@ -6,9 +6,9 @@
 #define DRIVER_SUPER_PARAMS_H_
 
 #undef CDS_STREAM_MAX
-#undef CDS_IF_RX_BUFFER_SIZE
+#undef CDS_HAL_RX_BUFFER_SIZE
 
 #define CDS_STREAM_MAX           (3)
-#define CDS_IF_RX_BUFFER_SIZE    (1024)
+#define CDS_HAL_RX_BUFFER_SIZE    (1024)
```

Comment:
これってスクリプトでsedするとき，1スペース除いたりできる？
つまり
```suggestion
#define CDS_HAL_RX_BUFFER_SIZE   (1024)
```

Reply:
できるけど，そういう特殊対応をあんまり自動でやるべきではないと思う

Reply:
まあ，そらそうか．PRわけて手でしゅしゅっとやるか

OKです．